### PR TITLE
add september blog post

### PR DESCRIPTION
**Pull request by dholbach**
Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Comment by dholbach** [comment]
I wasn't sure what to use as "featured image" in https://deploy-preview-1176--fluxcd.netlify.app/blog/ - if anyone has suggestions, let me know.

**Comment by makkes** [comment]
> I wasn't sure what to use as "featured image" in https://deploy-preview-1176--fluxcd.netlify.app/blog/ - if anyone has suggestions, let me know.

Maybe an icon comprised of the [OCI logo](https://github.com/opencontainers) and the Flux logo?

Or a sunset (as a symbol for the Flux v1 retirement).

**Comment by dholbach** [comment]
Are we generally good with the legacy part?

File: content/en/blog/2022-10-04-september-update/index.md [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+
+11:00 - 12:30 KubeCon: [Flux
+ContribFest](https://sched.co/182QL)
+
+> Room 410 B
+
+16:55 - 17:30 KubeCon: [Flux Maturity, Feature, and Contrib
+Update](https://sched.co/182QX)
+
+> Paulo Gomes & Sanskar Jaiswal (Weaveworks)
```
Comment (aryan9600) [inline content/en/blog/2022-10-04-september-update/index.md:366]:
```suggestion
> Somtochi Onyekwere & Kingdon Barrett (Weaveworks)
```

File: content/en/blog/2022-10-04-september-update/index.md [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+9:45 - 10:15 GitOpsCon: [Toward Full Adoption of GitOps and Best
+Practices at RingCentral](https://sched.co/1AR8J)
+
+> Tamao Nakahara (Weaveworks) and Ivan Anisimov (RingCentral)
+
+11:10 - 11:40 GitOpsCon: [Simplifying Edge Deployments Using EMCO and
+GitOps](https://sched.co/1AR8V)
+
+> Igor DC & Adarsh Vincent Chittilappilly (Intel)
+
```
Comment (aryan9600) [inline content/en/blog/2022-10-04-september-update/index.md:299]:
```suggestion

11:40 - 12:10 Promethus Days: [Automate your SLO validation with Prometheus & Flagger](https://sched.co/1AsMU)

> Sanskar Jaiswal & Kingdon Barrett (Weaveworks)

```

**Review by aryan9600 — CHANGES_REQUESTED** [review]

**Comment by scottrigby** [comment]
this is looking good btw!

**Comment by dholbach** [comment]
I added a flux+oci logo - if somebody wants to improve it - please go ahead :-D

**Review by phillebaba — APPROVED** [review]
LGTM!

**Comment by makkes** [comment]
The feature image is too heavily cropped in my view:


I changed it a little so that not as much content is cropped off of it:



I can't directly contribute to this PR as it comes from a fork but here's a commit changing the feature image: https://github.com/fluxcd/website/blob/ab9bb9312960279cfedd600e9ea726cce9664693/content/en/blog/2022-10-04-september-update/flux%2Boci-featured.png

**Comment by dholbach** [comment]
Thanks a lot @makkes!

**Comment by dholbach** [comment]
This looks better now!

**Pull request merged**